worldedit-bukkit/build.gradle.kts

```diff
@@ -150,6 +150,8 @@ tasks.named<ShadowJar>("shadowJar") {
         // In tandem with not bundling log4j, we shouldn't relocate base package here.
         // relocate("org.apache.logging", "com.sk89q.worldedit.log4j")
         relocate("org.antlr.v4", "com.sk89q.worldedit.antlr4")
+        // GriefDefender provide flow-math
+        relocate( "com.flowpowered", "com.griefdefender.lib.flowpowered")
```

Comment:
If GD provides flowpowered at runtime, there's no need for us to shade it in and relocate it under a package we should not touch. Nor should we shade fourth party dependencies.

Reply:
It's not being shaded? Presumably GD shades and relocates, this this is the same requirement as using the p2 bundled kyori stuff

Reply:
flow-math is included in GriefDefenderAPI, If we don't relocate it, the method `claim.getGreaterBoundaryCorner()` can not work correctly.

Reply:
Sounds like something GD has to resolve rather than us working with hacky methods again.

> Presumably GD shades and relocates, this is the same requirement as using the p2 bundled kyori stuff

We do that there, because we actually work with adventure stuff and need to send messages and access them. The flowmath references have been removed.

Reply:
This is definitely an issue with GD's API that I'll have fixed in GD 2.1.0. I'll be including the relocated libraries in GD API 2.1.0 to match what is used at runtime so plugins no longer have to worry about relocation.
@ahdg6 Please update this PR after GD 2.1.0 is released.

Reply:
@NotMyFault this can probably be merged with a note on the relocation? Rather than waiting for GD 2.1.0. Unless 2.1.0 is not long to be released @bloodmc ?

Reply:
```toml
griefdefender = "2.0.0-20211206.072436-12"
```
needs to be changed to a MAJOR.MINOR.PATCH version, otherwise renovate can't pick it up according to our configuration.